src/pug/includes/body.pug

```diff
@@ -82,6 +82,14 @@
 						.flex
 							a.btn(href=swag.reference target="_blank") Check it out
 
+			.no-results(style="display: none")
+				h3 Oh no! There are no opportunities that match those filters. Contribute one on Github.
+				.flex
+					a.github(href="https://github.com/swapagarwal/swag-for-dev" target="_blank")
```

Comment:
Rather than duplicate the svg, can we use [svg use](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/use) since we also have a github icon in the footer?

Reply:
I saw there was the Github icon svg in the main section too so I've added a sprite at the top of the body and used svg use to duplicate the path node to each element instance.